styles_src/default.scss

```diff
@@ -56,16 +54,26 @@ $image-margin: 7px;
     }
   }
 
-	&-item {
+  &-item {
     height: $row-height;
     margin: 0 0 7px 0;
 
+    position: relative;
+
+    &-group {
+      height: 100%;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      width: calc(100% - (#{$image-size} + #{$image-margin}));
+    }
+
     &-bar {
+      width: 100%;
       float: left;
```

Comment:
removing this `float: left;` would make the padding-top value for the majority group more clear as the bar would not float out of context for the elements appearing in the DOM later. Also: this is probably a leftover from a previous design? At least I do not see why it should float, but I could very well miss something.

Reply:
yes indeed, and I was wondering why I have to add the margin on majority line and arrow. Now I know and will remove these stuff.

Reply:
very nice catch, that way I could remove the height of each candidate item as well which also had bad side effects on the margin